src/pages/index.js

```diff
@@ -32,6 +52,43 @@ const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardContainer = addCardModal.querySelector("#add-card-container");
 const addCardSubmitButton = addCardModal.querySelector(".modal__button");
 
+/* -------------------------------------------------------------------------- */
+/*              variables that belong to the confirm delete popup             */
+/* -------------------------------------------------------------------------- */
+
+const confirmDeleteModal = document.querySelector("#confirm-delete-modal");
+const confirmDeleteModalContainer =
+  confirmDeleteModal.querySelector(".modal__container");
+const confirmDeleteModalForm = confirmDeleteModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms